### ISSUE TITLE
Switching users doesnt destroy mqtt listeners causing pipeline errors

### ISSUE THREAD

**Issue by rssk** [MRN-Code/coinstac#1552]
## Task Description
As title
* logging out and logging in as a different user causes all pipelines to hang 
* look in `coinstac-pipeline` or perhaps how the ui deals with the pipeline object via `coinstac-core`, listeners are seeming to still hang around. This can be verified looking at the local mosq logs

**Comment by dylanmartin** [comment]
The last I recall, this was happening because an instance of pipeline-manager or something similar didn't actually get destroyed and recreated on user switch.

**Comment by rssk** [comment]
Correct, the mqtt connection was still alive when the user switched, which should not be happening. The whole core-init chain in `main/index` is the place all of that stuff happens

**Comment by dylanmartin** [comment]
@rssk is this fixed?
